docs/code-of-conduct-response.rst

```diff
@@ -1,76 +1,121 @@
+To provide a number of examples, these are situations where immediate removal from a space is likely appropriate:
+
+* Repeated violations that resulted in a warning from staff
+* Continuing to harass after any "No" or "Stop" instruction
+* A pattern of harassing behavior, with or without warnings
+* A single serious offense (e.g., punching or groping someone)
+* A single obviously intentional offense (e.g., taking up-skirt photos)
+
+The response team should not involve venue security or law enforcement, unless they have consent from the affected person(s), or if the safety risk is so significant that there is no other reasonable option. Venue security can be involved to ensure a person can not re-enter the building. However, details of the incident or the reporter must not be shared.
+
+For online spaces, a response must be sent within one week to the person who filed the report with either a resolution or an explanation of why the situation is not yet resolved. For conferences and meetups, the ideal response time is typically a few hours at most, though the specific circumstances of a report may require faster action or more investigation.
+
+Informing the reporter
+~~~~~~~~~~~~~~~~~~~~~~
+
+After the resolution is complete, the reporter should be informed of the action taken by the response team, and the reasoning behind this.
 
 Public statements
 ~~~~~~~~~~~~~~~~~
-As a general rule, conference staff should not make any public statements about the behavior of individual people during or after the conference. After each conference, a CoC transparency report will be published with anonymized information about any violations that might have occured.
 
-This report should be handled with care not to divulge personally identifying information about victims, reporters, and violators, and should serve as a means to ensure that attendees will be comfortable reporting harrasment and that our community will be kept accountable for supporting and encouraging safe spaces.
+As a general rule, conference staff should not make any public statements about the behaviour of individual people during or after the conference. An exception to this are situations that happened in a fairly public context, because attendees may otherwise think no action was taken, eroding trust in the Code of Conduct process. The response team will decide together whether to make a public statement, and if so, in how much detail. This should not be decided unilaterally.
+
+After each conference, a Code of Conduct transparency report will be published by the response team with anonymised information about any violations that might have occurred.
+
+Any public statements should be handled with care not to divulge personally identifying information about anyone affected, and should serve as a means to ensure that attendees will be comfortable reporting suspected CoC violations and that our community will be kept accountable for supporting and encouraging safe spaces.
```

Comment:
we can probably link to previous transparency reports here, where we had incidents, or maybe also to other confs' reports, so that people can reference wording and format